src/components/scenes/RequestScene.tsx

```diff
@@ -370,10 +371,10 @@ export class RequestSceneComponent extends React.Component<Props, State> {
               />
             )}
           />
-          <TouchableOpacity onPress={this.handleAddressBlockExplorer}>
+          <TouchableOpacity disabled={addressExplorerDisabled} onPress={this.handleAddressBlockExplorer}>
             <View style={styles.rightChevronContainer}>
               <EdgeText>{selectedAddress?.label ?? s.strings.request_qr_your_wallet_address}</EdgeText>
-              <IonIcon name="chevron-forward" size={theme.rem(1.5)} color={theme.iconTappable} />
+              {!addressExplorerDisabled ? <IonIcon name="chevron-forward" size={theme.rem(1.5)} color={theme.iconTappable} /> : null}
```

Comment:
Optional: I generally like to put the `null` case first, so `addressExplorerDisabled ? null : <IonIcon .../>` would be easier to read.